internal/migrate/import.go

```diff
@@ -97,22 +77,7 @@ func (m *Importer) Import(ctx context.Context, data *types.Org) error {
 
 	m.Tracer.Stop("create organisation secrets [done]")
 
-	m.Tracer.Start("create connector %s", m.ScmType)
```

Comment:
Do we want to remove this functionality? Should it not be, if connector not passed in, we create?

Reply:
agreed, I think we should keep connector creation (and secret creation).  If the customer already created a connector we can skip this step, but we should not remove entirely, since we still want to eventually provide a touchless migration experience with no setup required